src/libs/merge/kdbmerge.c

```diff
@@ -220,6 +369,35 @@ static char * strremove (char * string, const char * sub)
 	return string;
 }
 
+static Key * prependStringToKeyName (Key * key, const char * string, Key * informationKey)
```

Comment:
Please have at least short docu also for internal stuff (e.g. why is /root special, preconditions, etc.)

Reply:
Sure, but it will take some time. I didn't really write this function, just extracted it when I refactored another method.

Reply:
Yeah, it is difficult to get used to other code. You are doing a great job here.